### PR TITLE
test: add test (shouldn't overflow when coloring already colored large text)

### PR DESCRIPTION
**Pull request by Henry-Hong**
### Description
- related: #64 

Thanks to above PR, there's no more call stack overflow when coloring already colored long string.

I added some tests to compensate a bit.

I wanted to add tests to the Vite project, but it was pretty tricky, so I made a PR here.

Following's related PR on Vite project.
- https://github.com/vitejs/vite/pull/17729

### As-Is (Test fail)
image

### To-Be (Test success)
image

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -156,6 +156,15 @@ test("non-string input", () => {
 	assert.equal(pc.red(Infinity), FMT.red[0] + "Infinity" + FMT.red[1])
 })
 
+test("shouldn't overflow when coloring already colored large text", () => {
+  try {
+    pc.blue(pc.red("x").repeat(10000))
+    assert(true)
+  } catch {
```
Comment (alexeyraspopov) [inline tests/test.js:163]: Omitting `(error)` only works in newer Node.js versions. Could you please add `(error)` so the rest of pipelines would go green? Thanks

Comment (Henry-Hong) [reply tests/test.js]: Oops! 
Didn't check the CI tests.
Thank you! 

Now everything is green :)
